FlowCrypt/Functionality/Services/Key Services/InMemoryPassPhraseStorage.swift

```diff
@@ -70,7 +61,6 @@ final class InMemoryPassPhraseStorage: PassPhraseStorageType {
 protocol InMemoryPassPhraseProviderType {
     var passPhrases: Set<PassPhrase> { get }
     func save(passPhrase: PassPhrase)
-    func removePassPhrases(with objects: PassPhrase)
```

Comment:
`removePassPhrases` - I suppose this would be used during logout? Or is there some other way to throw away all pass phrases in memory? (also in the future we'll have a button for forgetting pass phrases)

Reply:
Before deleting I checked where `removePassPhrases` is used and it was used only in 1 place - https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCrypt/Functionality/Services/Key%20Services/PassPhraseService.swift#L86

So after removing `EncryptedStorage` I removed this method too, as it wasn't used anywhere else.
It seems currently we're not deleting passphrases from memory on logout. I'll rollback this change and call `removePassPhrases` on logout.

Reply:
I added back `EncryptedStorage`, but now it uses passphrases stored in `KeyInfo` table.

For deleting in-memory passphrases I created separate task https://github.com/FlowCrypt/flowcrypt-ios/issues/587, as currently `UserAccountService` doesn't have direct reference to `InMemoryPassPhraseStorage` and I'll think of better handling for this case.